backend/curation/api/v1/curation/collection_versions/collection_version_id/actions.py

```diff
@@ -14,10 +18,13 @@ def get(collection_version_id: str):
     Fetch the full metadata of a published CollectionVersion, including its datasets
     """
     validate_uuid_else_forbidden(collection_version_id)
-    version = get_business_logic().get_collection_version(CollectionVersionId(collection_version_id))
-    if version is None:
-        raise NotFoundHTTPException()
-    if version.published_at is None or version.canonical_collection.tombstoned is True:
-        raise ForbiddenHTTPException()
+    try:
+        version = get_business_logic().get_published_collection_version__discover_api(
+            CollectionVersionId(collection_version_id)
+        )
+    except PublishedCollectionVersionNotFoundException as e:
```

Comment:
What is the benefit of pushing this error logic into `get_business_logic().get_published_collection_version__discover_api`?